routes/assetsRoute.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
```

Comment:
Oh, add and removed...perhaps i shouldn't be reviewing based on commits, but full file changes instead? Happy to do either, just was trialing reviewing by commit since I haven't been able to do that for years after being on Phab.

Perhaps you should have just left these two commits out? Sure that requires rebasing, but in the end the history is actually cleaner for it.

Reply:
oh, you're taking this commit by commit. Ok. I'll keep that in mind and just rebase/squash before putting the reviews up

Reply:
Thx